less/quality.less

```diff
@@ -0,0 +1,42 @@
+@font-face {
+  font-family: 'icomoon';
+  src:  url('../img/fonts/icomoon.eot?r9fnjw');
```

Comment:
Are you sure of this filenames?
The files are rather named ``quality.oet``, ``quality.ttf``, etc.